internal/cli/up.go

```diff
@@ -83,7 +83,7 @@ local resources--i.e. those using serve_cmd--are terminated when you exit Tilt.
 	cmd.Flags().StringVar(&updateModeFlag, "update-mode", string(buildcontrol.UpdateModeAuto),
 		fmt.Sprintf("Control the strategy Tilt uses for updating instances. Possible values: %v", buildcontrol.AllUpdateModes))
 	cmd.Flags().StringVar(&c.traceTags, "traceTags", "", "tags to add to spans for easy querying, of the form: key1=val1,key2=val2")
-	cmd.Flags().BoolVar(&c.hud, "hud", true, "If true, tilt will open in HUD mode.")
+	cmd.Flags().BoolVar(&c.legacy, "legacy", true, "If true, tilt will open in Legacy mode.")
```

Comment:
I played around with this and this behavior doesn't feel right to me. see my comment here:
https://app.clubhouse.io/windmill/epic/7733/rename-hud-option-for-tilt-1-0-to-emphasize-legacy-status-of-hud#activity-8636

Reply:
Will follow up on that!